src/components/amountInput.tsx

```diff
@@ -104,10 +104,10 @@ export default class AmountInput extends React.Component<
   public render() {
     const { onColorChange, value, width = 250, plasma, ...rest } = this.props;
     const balance = plasma ? this.token.plasmaBalance : this.token.balance;
-
+    const enumerableNft = this.token.isNft && balance && balance[0];
```

Comment:
`Array.isArray(balance)`?

Reply:
so check for `balance[0]` actually distringuish between enumerable and non-enumerable here. For the latter it will be undefined. Yet both are arrays